index.html

```diff
@@ -134,9 +134,35 @@ <h3 class="title-1">Multi-Post Stories</h3>
         <li class="ruby">Ruby</li>
       </ul>
 
-      <button class="see-1" type="button">See Project</button>
+      <button data-modal-target="#modal-mobile" class="see-1" type="button">See Project</button>
+      <div class="mobile-modal" id="modal-mobile">
+        <div class="mobile-modal-header">
+          <h2 class="mobile-modal-title">Multi - Post stories</h2>
+          <button data-close-button class="close-mobile-modal" type="button" ><img class="close-image" src="./images/close-icon.png" alt="Close icon Mobile modal"/></button>
+        </div>
+        <div class="mobile-modal-body">
+          <img class="mobile-modal-image" src="./images/modal-mobile.png" alt="Project Image on mobile device"/>
+          <img class="desktop-modal-image" src="./images/modal-desktop.png" alt="Project Image on desktop device">
+          <p class="mobile-modal-text">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting  ever since the 1500s, when an unknown printer took a galley of type veris lapoa todoe.</p>
+          <p class="desktop-modal-text">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scramble.</p>
+
+          <ul class="mobile-modal-lang">
+            <li class="liner">|</li>
+            <li class="html">html</li>
+            <li class="liner-2">|</li>
+            <li class="boot">Ruby on rails</li>
+            <li class="liner-3">|</li>
+            <li class="ruby">css</li>
+            <li class="liner-4">|</li>
+            <li class="github-text">Github</li>
+          </ul>
+          <button class="live-button" type="button">See Live <img class="live-icon" src="./images/Icon.png" alt="Live link icon"></button>
+          <button class="src-button" type="button">See Source <img class="src-icon" src="./images/Vector(1).png" alt="Github link icon"></button>
+        </div>
+      </div>
+
+      <div id="overlay"></div>
 
```

Comment:
Hello @roshan-bajgain . 👋 I Really appreciate your review. I would like to ask for more clarification on the matter because when I use the syntax above it returns undefined when printing out the information. When I use this syntax it actually works: this is the syntax I used;

![syntax](https://user-images.githubusercontent.com/83514256/179865186-7a9eaba8-2a11-4fb2-a7d9-0fa8c75bdbfb.PNG)

I will really appreciate your clarification

Reply:
Hello @David-Kasilia, You have all the code of the project section in HTML I would suggest you copy all the project sections and add them dynamically with an array using Javascript.

Reply:
@David-Kasilia, You are doing it a correct way, after listing in an object render dynamically. You can see example for reference.
`const popUpProject = (popUpdata) => {
     <img src="....." id="popUp-Close">
     <h2 class="card-title">${popUpdata.title}</h2>
         <div class="work-detail-box">
             <ul>
                 <li class=" work-detail canopy">CANOPY </li>
                 <li class="work-detail work-text det">Back End Dev </li>
                 <li class="work-detail work-text det">2015</li>
             </ul>
         </div>`

Reply:
Hello, @roshan-bajgain thank you for getting back to me. I appreciate it. When I implement the above method it returns some of the array objects ${name} instead of printing out the actual value of the array. any ways to fix that issue?